cime_config/tests.py

```diff
@@ -226,9 +226,9 @@
 
     "fates" : {
         "tests" : (
-            "ERS_Ld9.1x1_brazil.IELMED",
-            "ERS_D_Ld9.1x1_brazil.IELMED",
-            "SMS_D_Lm6.1x1_brazil.IELMED",
+            "ERS_Ld9.1xFATESbrazil.IELMFATES",
```

Comment:
Is this a typo?

Reply:
yes, nice catch @glemieux , will correct